src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
```

Comment:
It's good that you've specified a font-family for your body text. However, don't forget to use fallback fonts - an alternative font-family in case the main one doesn't work. For instance, `font-family: Roboto, Arial, sans-serif;`.